specification/ai/Azure.AI.Unified/main.tsp

```diff
@@ -0,0 +1,70 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "./agents/routes.tsp";
+import "./agents/threads/routes.tsp";
+import "./agents/messages/routes.tsp";
+import "./agents/runs/routes.tsp";
+import "./agents/run_steps/routes.tsp";
+import "./agents/files/routes.tsp";
+import "./agents/streaming/events.tsp";
+import "./agents/vector_stores/routes.tsp";
+import "./agents/vector_stores/files/routes.tsp";
+import "./agents/vector_stores/file_batches/routes.tsp";
+import "./connections/routes.tsp";
+import "./evaluations/routes.tsp";
+import "./datasets/routes.tsp";
+import "./models/routes.tsp";
+import "./indexes/routes.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+#suppress "@azure-tools/typespec-azure-core/casing-style"
+namespace Azure.AI {
+
+}
+
+@useAuth(
+  OAuth2Auth<[
+    {
+      type: OAuth2FlowType.implicit,
+      authorizationUrl: "https://login.microsoftonline.com/common/oauth2/v2.0/authorize",
+      scopes: ["https://management.azure.com/.default"],
+    }
+  ]>
+)
+@service({
+  title: "Azure AI Data Plane",
+})
+@server(
+  "{endpoint}/projects/{projectName}",
+  "Azure AI Data Plane APIs",
+  {
+    @doc("""
+      Global endpoint in the form of: https://[hub-id]/api.ai.azure.com
+      """)
+    endpoint: url,
+
+    @doc("The name of the AI project.")
+    projectName: string,
+  }
+)
+@versioned(Versions)
+namespace Azure.AI.Unified {
+  @doc("Azure AI API versions")
+  enum Versions {
+    @doc("Azure AI API version 2024-07-01-preview.")
+    @armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.v3)
+    @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
+    @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+    `2024-07-01-preview`,
```

Comment:
Should this change to `2025-05-01-preview`?

Reply:
I updated it for now, though I will confirm with the team on the supported API versions sometime this week

Reply:
Based on Billy's new document, we need to list two versions: "latest" and "2025-05-01". And make sure the SDK uses "2025-05-01". Not sure how to specify which of the versions the SDK should use. I can look into it later.